.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,35 @@
+name: Feature Request
+description: Request a new feature for this SDK?
```

Comment:
Just to confirm, do we use term SDK for this repo?

Reply:
Fixed.